packages/plugin-core/src/commands/CreateDailyJournal.ts

```diff
@@ -1,15 +1,43 @@
-import { ConfigUtils, DendronError, VaultUtils } from "@dendronhq/common-all";
+import {
+  ConfigUtils,
+  DailyJournalTestGroups,
+  DendronError,
+  genUUID,
+  JournalConfig,
+  NoteUtils,
+  SchemaCreationUtils,
+  SchemaToken,
+  SchemaUtils,
+  VaultUtils,
+  _2022_05_DAILY_JOURNAL_TEMPLATE_TEST,
+} from "@dendronhq/common-all";
+import { SegmentClient, vault2Path } from "@dendronhq/common-server";
 import _ from "lodash";
+import * as fs from "fs-extra";
+import path from "path";
+import * as vscode from "vscode";
+import { PickerUtilsV2 } from "../components/lookup/utils";
 import { DENDRON_COMMANDS } from "../constants";
 import { IDendronExtension } from "../dendronExtensionInterface";
+import { ExtensionProvider } from "../ExtensionProvider";
 import { JournalNote } from "../traits/journal";
 import {
   CommandOpts,
   CreateNoteWithTraitCommand,
 } from "./CreateNoteWithTraitCommand";
+import { VSCodeUtils } from "../vsCodeUtils";
+import { MetadataService } from "@dendronhq/engine-server";
+
+export type ExecuteData = {
```

Comment:
nit: can you rename this to something like `CreateDailyJournalData` or something specific to this command type?